crates/rooch-rpc-client/src/client_config.rs

```diff
@@ -15,7 +15,7 @@ use serde_with::serde_as;
 use std::fmt::{Display, Formatter, Write};
 
 pub const DEFAULT_EXPIRATION_SECS: u64 = 30;
-pub const ROOCH_TEST_NET_URL: &str = "https://seed-testnet.rooch.network/";
+pub const ROOCH_TEST_NET_URL: &str = "https://dev-seed.rooch.network/";
```

Comment:
Const name should be consistent with value. Maybe you can add a new Const `ROOCH_DEV_NET_URL`

Reply:
Yes. It needs to be changed but at this time the dev net isn't working.